src/user/users.controller.ts

```diff
@@ -16,4 +27,41 @@ export class UsersController {
     await this.usersService.create(mockUser);
     return true;
   }
+  @Get()
+  @HttpCode(HttpStatus.OK)
+  async getUsers(): Promise<CreateUserDto[]> {
+    return this.usersService.findAll();
+  }
+
+  @Get(':id')
+  @HttpCode(HttpStatus.OK)
+  async getUser(@Param() params) {
+    return this.usersService.findById(params.id);
+  }
+
+  @Post()
```

Comment:
I think you should ignore `user.passwaord` before return data to client?

Reply:
Ok ,I will fix this